cloudfoundry/resource_cf_service_instance.go

```diff
@@ -82,9 +92,20 @@ func resourceServiceInstanceCreate(d *schema.ResourceData, meta interface{}) (er
 	if id, err = sm.CreateServiceInstance(name, servicePlan, space, params, tags); err != nil {
 		return err
 	}
-	session.Log.DebugMessage("New Service Instance : %# v", id)
+	stateConf := &resource.StateChangeConf{
+		Pending:      resourceServiceInstancePendingStates,
+		Target:       resourceServiceInstanceSucceesStates,
+		Refresh:      resourceServiceInstanceStateFunc(id, "create", meta),
+		Timeout:      d.Timeout(schema.TimeoutCreate),
+		PollInterval: 5 * time.Second,
```

Comment:
If I do remember well Cloud Controller is polling every 30 seconds maybe poll every 5 seconds is a bit too much

Reply:
@ArthurHlt considering #107, wouldn't setting a 30s polling interval delay all deletes by 30s?
@janosbinder proposed setting a 5s min timeout, and a 30s refresh period.